suricata-ocsf/package.yaml

```diff
@@ -117,150 +98,120 @@ pipelines:
     disabled: false
     definition: |
       // tql2
+      let $rcode_id = {
+        NOERROR: 0,
+        FORMERROR: 1,
+        SERVERROR: 2,
+        NXDOMAIN: 3,
+        NOTIMP: 4,
+        REFUSED: 5,
+        YXDOMAIN: 6,
+        YXRRSET: 7,
+        NXRRSET: 8,
+        NOTAUTH: 9,
+        NOTZONE: 10,
+        DSOTYPENI: 11,
+        BADSIG_VERS: 16,
+        BADKEY: 17,
+        BADTIME: 18,
+        BADMODE: 19,
+        BADNAME: 20,
+        BADALG: 21,
+        BADTRUNC: 22,
+        BADCOOKIE: 23,
+      }
+      let $rcode = {
+        NOERROR: "NoError",
+        FORMERROR: "FormError",
+        SERVERROR: "ServError",
+        NXDOMAIN: "NxDomain",
+        NOTIMP: "NotImp",
+        REFUSED: "Refused",
+        YXDOMAIN: "YXDomain",
+        YXRRSET: "YXRRSet",
+        NXRRSET: "NXRRSet",
+        NOTAUTH: "NotAuth",
+        NOTZONE: "NotZone",
+      }
       subscribe "suricata"
       where @name == "suricata.dns"
-      this = { event: this }
+      this = { suricata: this }
+      // === Classification ===
       // TODO: Proper request/response event splitting.
-      has_query = false
-      has_response = false
-      if event.dns.rrname != null {
+      ocsf.has_query = false
+      ocsf.has_response = false
```

Comment:
Note that this should not be part of the resulting OCSF event, as these are just temporary values used below.